includes/languages/english/extra_definitions/ask_a_question_defines.php

```diff
@@ -7,5 +7,3 @@
  * @TODO consider merge into core files 
  */
 define('ASK_A_QUESTION', 'Ask a question about this product');
```

Comment:
I guess this single remaining constant should be moved into `english.php` and this file deleted.